blog/src/pages/changelog-weekly.js

```diff
@@ -37,40 +37,37 @@ const ChangelogWeekly = ({ data }) => {
     fetchData();
   }, []);
 
-  useEffect(() => {
-    const rate = (version, rating) => {
-      let issue = "";
-      if (rating <= 0) {
-        issue = prompt("Please provide issue number from our JIRA causing trouble:", "");
+  const rate = useCallback((version, rating) => {
```

Comment:
Hi @Vandit1604, I have ussd the `useCallback` hook here instead of the `useEffect` since this should be a callback and not something to be updated, both here and for the LTS changelog. Please be careful not to use the `useEffect` here since that would not be suitable. Now we do not need
```
window.rate = rate; // Expose rate function globally.  
```